app/models/conference.rb

```diff
@@ -595,8 +595,38 @@ def registration_limit_exceeded?
     registration_limit > 0 && registrations.count >= registration_limit
   end
 
+  # Return a different html color for every new track. Consecutive talks have
+  # highly different colors.
+  def next_track_color
```

Comment:
But if we put then together we lose the fact that following tracks has highly different colors as we could create other things in the middle. It is not that important that a track and a difficult level have the same color but it is important that we create all of them with the same index).

I can call `next_color_for_collection(collection, start_index)` directly in the controller and eliminate the other 3 methods, would that be fine?